nvfuser/__init__.py

```diff
@@ -33,6 +33,20 @@
 logger = logging.getLogger("nvfuser")
 
 
+# Register automatic serialization of Nvfuser cache hierarchy and cuda kernels.
+def enable_automatic_serialization():
+    import atexit
+
+    atexit.register(_C.serialize)
+
+
+# Unregister automatic serialization of Nvfuser cache hierarchy and cuda kernels.
+def disable_automatic_serialization():
```

Comment:
This thing isn't used.

Reply:
This is to give users the option to disable automatic serialization in Thunder.